netlify/remix-compute
# feat: moved to Remix routing v2

## Description

Configures the default Remix config to use the new v2 routing logic. With these changes, the index route is now renamed to `_index.tsx` from `index.tsx`.

## Related Tickets & Documents



- Related Issue #
- Closes #60

## QA Instructions, Screenshots, Recordings

1. Navigate to https://deploy-preview-61--remix-edge.netlify.app/ in the deploy preview site. The page loads

![CleanShot 2023-03-13 at 14 27 52](https://user-images.githubusercontent.com/833231/224795971-65d2c402-4e83-4238-9538-f4eb50585fb3.png)

2. Navigate to https://deploy-preview-61--remix-edge.netlify.app/about in the deploy preview site. The page loads. 

![CleanShot 2023-03-13 at 14 29 09](https://user-images.githubusercontent.com/833231/224796167-a2747099-bd10-4545-9230-a2ea12562a15.png)

For us to review and ship your PR efficiently, please perform the following steps:

      ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a
      typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
      guide and passes our tests.

**A picture of a cute animal (not mandatory, but encouraged)**
